apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -104,12 +125,14 @@ export const deck: DbCard[] = [
 		id: '7',
 		type: 'field',
 		name: 'Sacred Pool',
-		description: 'Whenever you attack with a spell from the blue stack, you heal 10HP.',
+		description: 'Whenever players attack with the same color, they heal 10 HP.',
```

Comment:
can we call players wizards? it'd be more in character.

for clarity:
"they *each* heal 10HP"
or
"both heal 10HP"

Reply:
but i sort of preferred the older version of this card, as it does sort of bias users to using blue because they heal a little bit. it feels more meta than "lets try to both hit blue and heal!"

Reply:
Honestly we can go with either, these could even be two different card. But I wanted to lean into the RPS system. There is a 1 in 3 chance you draw, and when that happens, nothing comes from it. So it would be a gentle mechanic that IF you happen to match, then something happens. (I want to create more mechanics like this)

Also for context: The first intention of the card was more with my suggestion. The change you made yesterday, I just wanted us to have the example mechanic so didn't say anything. But again, it's a card, we can just make two cards if we are really passionate about letting players heal with blue.

Reply:
yeah thats for sure! i just dont really see the point in both healing if both play blue, it's just making games a little bit longer, but I guess it doesn't hurt? someone could have "whenever you heal you draw a card" or something